### ISSUE TITLE
Change the default shebang in docs to use /usr/bin/env

### ISSUE THREAD

**Issue by guillep** [guillep/Scale#12]
Like
# ! /usr/bin/env scale

This will make script use the scale version that is found in the path. Like that people may have different installations and use the one they want given their current configuration.

**Comment by sbragagnolo** [comment]
Good idea! It goes for next version, since it will use this as excuse for changing the code browser :). (Now it checks the shebang to know what scriptrunner to use as container on the browser.)

**Comment by rajula96reddy** [comment]
Hi, should it be like user specifying which version to run, before running the command?

**Comment by guillep** [comment]
Maybe. A user wanting to use a specific version of scale write instead:

```bash
!/usr/bin/env scale
```

```bash
!/usr/bin/env scale-2.7
```

```bash
!/usr/bin/env scale-3
```

Of course that would require having available in `PATH` all scale versions. That could be done "à la" linux by adding symlinks of course

**Comment by rajula96reddy** [comment]
Maybe the scale bash script can be edited and user can be prompted for version and that specific version can be invoked. And apart from this, what more can be done for this?

**Comment by guillep** [comment]
The main intention of this issue is to update the documentation. New features or the scale GUI support should be discussed in separate issues please.

**Comment by rajula96reddy** [comment]
Sorry.! My fault. Will do it in a new issue.

**Comment by guillep** [comment]
Thanks, in any case, I think updating the readme would be already a good contribution.

**Comment by rajula96reddy** [comment]
Thank you. I will try to contribute to the readme.
